inscriptions/ISic000386.xml

```diff
@@ -185,21 +190,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141196" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR141196" xml:lang="la" resp="#JH">
                 <ab>
                     <lb n="1"/>
-                    <gap reason="lost" extent="unknown" unit="character"/><orig>NISTER</orig>
-                    <gap reason="lost" extent="unknown" unit="character"/>
+                        <gap reason="lost" extent="unknown" unit="character"/><orig>nister</orig>
+                        <gap reason="lost" extent="unknown" unit="character"/>
                     <lb n="2"/>
-                    <gap reason="lost" extent="unknown" unit="character"/><orig>DAER</orig>
+                        <gap reason="lost" extent="unknown" unit="character"/><orig>daer</orig>
```

Comment:
notwithstanding the lack of dots in CIL, I think one has to put a gap at the end of line 2 as well, given the incoherence of the letters